locales/it/json.json

```diff
@@ -247,13 +247,13 @@
     "Email Address": "Indirizzo Email",
     "Email Addresses": "Indirizzi Email",
     "Email Password Reset Link": "Invia link di reset della password",
-    "Email Verification": "Email Verification",
+    "Email Verification": "Verifica Email",
     "Enable": "Abilita",
-    "Encrypted environment file already exists.": "Encrypted environment file already exists.",
-    "Encrypted environment file not found.": "Encrypted environment file not found.",
+    "Encrypted environment file already exists.": "Il file di ambiente criptato esiste già.",
+    "Encrypted environment file not found.": "File di ambiente criptato non è stato trovato.",
```

Comment:
Change with:

"Non è stato trovato il file di ambiente criptato."

Reply:
You can speed up this process a bit 🙂

![image](https://github.com/user-attachments/assets/d9389ad8-ca1d-43fe-8ea9-d19c32947bfb)

```suggestion
    "Encrypted environment file not found.": "Non è stato trovato il file di ambiente criptato.",
```